lib/interpreters/split_clustering.rb

```diff
@@ -1203,15 +1207,17 @@ def add_corresponding_entity_skills(entity, vrp)
         case entity
         when :vehicle
           vrp.services.each{ |service|
-            service.skills.insert(0, vrp.vehicles.first.id)
+            service.skills.insert(0, "vehicle_partition_#{vrp.vehicles.first.id}".to_sym)
+          }
+          vrp.vehicles.each{ |v|
+            v.skills.each{ |set| set << "vehicle_partition_#{vrp.vehicles.first.id}".to_sym }
```

Comment:
Same comment here, it is not possible to have this skill from original skills, neither from partition by vehicle twice because partition by vehicle generates cluster with only one vehicle so if we call vehicle partition twice, there will be no clustering second time.